frontend/packages/core/src/Theme/theme.tsx

```diff
@@ -36,41 +40,49 @@ declare module "@mui/material/styles" {
   interface ThemeOptions {
     clutch: {
       useWorkflowLayout: boolean;
-      spacing: {
-        none: number;
-        xs: number;
-        sm: number;
-        base: number;
-        md: number;
-        lg: number;
-        xl: number;
-      };
       layout: {
         gutter: string;
       };
     };
   }
 }
 
+// `8` is the default scaling factor in MUI, so we define and use it
+// to allow predictability using a `number` value
+// https://v5.mui.com/material-ui/customization/spacing/
+const DEFAULT_SCALING_FACTOR = 8;
+
+const CLUTCH_CUSTOM_SPACING = {
+  xs: "4px",
+  sm: "8px",
+  base: "16px",
+  md: "24px",
+  lg: "32px",
+  xl: "40px",
+};
+
 // Create a Material UI theme is propagated to all children.
 const createTheme = (variant: ThemeVariant, useWorkflowLayout: boolean): MuiTheme => {
   return createMuiTheme({
     colors: clutchColors(variant),
     palette: palette(variant),
-    // `8` is the default scaling factor in MUI, we are setting it again to make it explicit
-    // https://v5.mui.com/material-ui/customization/spacing/
-    spacing: 8,
+    spacing: (...args) => {
+      const spacingValues = args.map(value => {
+        if (typeof value === "number") {
+          return `${value * DEFAULT_SCALING_FACTOR}px`;
+        }
+
+        if (typeof value === "string" && CLUTCH_CUSTOM_SPACING[value] !== undefined) {
+          return CLUTCH_CUSTOM_SPACING[value];
+        }
+
+        return value;
+      });
+
+      return spacingValues.join(" ");
+    },
     clutch: {
       useWorkflowLayout,
-      spacing: {
```

Comment:
Maybe keep these for usage elsewhere?, like `<Grid padding={theme.clutch.spacing.md} />`
Using the values from `CLUTCH_CUSTOM_SPACING` so they are unified

Reply:
Done in [dfdf187](https://github.com/lyft/clutch/pull/3170/commits/dfdf187374d6f884f6d5384b507e3d9cdd0da6e4)